### ISSUE TITLE
Question: - Is there any way to download the ffmpeg from the python.

### ISSUE THREAD

**Issue by SohamTilekar** [python-ffmpegio/python-ffmpeg-downloader#6]
I want to install the ffmpeg from directly python code Instead of using the subprocess.
And also to Save the final binary in the other folder.

**Comment by tikuma-lsuhsc** [comment]
Yes, but you need to reverse-engineer from `__main__.py` to piece together various functions in this package for your needs. 

It was a deliberate decision for this library to not expose/document its inner mechanism so that the host site maintainers are properly credited. However, the project being open source, you are free to use any portion of the code as you wish (under GPL v2).

**Comment by FlorianEagox** [comment]
> I want to install the ffmpeg from directly python code Instead of using the subprocess. And also to Save the final binary in the other folder.

Also trying to do this, seems a bit of a silly requirement. Don't know why you can't just include in the license that the maintainers must be creditiedd in anything that uses this project. Kinda useless as just a standalone CLI thing

**Comment by tikuma-lsuhsc** [comment]
@FlorianEagox - 🤷 At this point, it is what it is. Even if I change my mind, I don't have enough time to work on it. If you propose a compelling API and implement it as a PR, I'll consider it.

**Comment by FlorianEagox** [comment]
word

**Comment by tikuma-lsuhsc** [comment]
One more thing. [static-ffmpeg](https://github.com/zackees/static_ffmpeg) package has evolved since its inception and I believe it downloads the ffmpeg from these servers. So, you can check it out.

**Comment by FlorianEagox** [comment]
Ooh okay, I'll give it a look, thanks a bunch

**Comment by tikuma-lsuhsc** [comment]
For those who wish to download & install in Python, you can study the source code in [`src/qt/wizard.py`](https://github.com/python-ffmpegio/python-ffmpeg-downloader/blob/5537966f46f0b839133194888e224a3d56715044/src/ffmpeg_downloader/qt/wizard.py#L1)

This is the easiest to understand how to use `_backend` submodule.
